svelte.config.js

```diff
@@ -1,12 +1,18 @@
-import adapter from '@sveltejs/adapter-auto';
+import auto from '@sveltejs/adapter-auto';
+import node from '@sveltejs/adapter-node';
 import preprocess from 'svelte-preprocess';
 import { mdsvex } from 'mdsvex';
 
 /** @type {import('@sveltejs/kit').Config} */
 const config = {
   extensions: ['.svelte', '.md'],
   kit: {
-    adapter: adapter(),
+    adapter: process.env.PREVIEW ? node() : auto(),
+  },
+  server: {
+    watch: {
+      usePolling: true
```

Comment:
Can you share why you added this?

Reply:
sveltekit uses chokidar under the hood to watch files and that doesn’t work in a docker container, so if we are to be able to develop in a docker container itself with okteto, to get HMR you need that polling config.